RELEASE_NOTES.md

```diff
@@ -1,5 +1,15 @@
 # ID Mapping Service release notes
 
+## 0.1.2
+* The MongoDB clients have been updated to the most recent version and the service tested against Mongo 7.
+* Added id mapping service container test in GHA
+* Added the mongo-retrywrites configuration setting in deployment.cfg.templ and deploy.cfg.example, defaulting to false.
+* Updated the docker-compose file to start a id mapping service server in test mode.
```

Comment:
```suggestion
* Updated the docker-compose file to start a id mapping service server.
```

the service doesn't have a test mode per se - the auth service is the only service I'm aware of that does that.

Reply:
:+1: